Userland/Utilities/cal.cpp

```diff
@@ -16,9 +16,10 @@
 #define ANSI_INVERT_OUTPUT "\e[7m"
 #define ANSI_RESET_OUTPUT "\e[0m"
 
-int const line_width = 70;
-int const line_count = 8;
-int const column_width = 22;
+// TODO: months are in reality 20-characters wide, but each line contains an extra erronous unneeded space at the end
+// so making this 20 breaks formatting a bit
+static int const month_width = 21;
+static int const year_width = month_width * 3 + strlen("  ") * 2; /* three months plus padding between them */
```

Comment:
Looks like you missed a `/* */` -> `//` style comment conversion (or fixup mishap?)

Reply:
“extra erronous uneeded” is just overstated; definitely the comment can be shortened.

(you could even declare month width as `20` chars — thatʼs factual — and declare another constant with a better name, e.g. `month_width_with_padding` or so, equal to `month_width + 1` to use it instead. you would comment only on that second constant then. just rough idea of how to deal with such issues with self-commenting code, i.e.e without using comments. not necessarily to be used, though!)

Reply:
@alimpfard Fixed, thanks!

@konradekk This imperfection is improved in the next commit 😄 I got rid of the extra `1` length there so the constant is now cleanly defined by doing `"01 02 03 04 05 06 07"sv.length();`

Reply:
👍